src/objects/page/page.scss

```diff
@@ -0,0 +1,29 @@
+/**
+ * Page container
+ *
+ * Attempts to fill the available area (assuming `height: 100%` is correctly
+ * applied to all parent elements) and keeps the footer at the bottom of that
+ * area.
+ *
+ * @see https://css-tricks.com/couple-takes-sticky-footer/#there-is-grid
+ */
+
+.o-page {
+  display: grid;
+  grid-template-rows: 1fr auto;
```

Comment:
Should we consider wrapping this layout in a min-height media query here? That would prevent the footer from obliterating the page content on devices with short screens, such as an iPhone in horizontal mode.

Reply:
Wait, no, I misunderstood. you set a min-height here, which means it's allowed to grow beyond the viewport. Sorry, my bad!

Reply:
I don't understand. Can you share a screenshot of the behavior you're describing?

When there isn't enough room for the body content and footer, the content will scroll as it normally would.

Reply:
![o-page-behavior](https://user-images.githubusercontent.com/69633/99465319-f6085200-28ee-11eb-9094-d40601258a7f.gif)